### PR TITLE
16: UI changes from BHS feedback (2/4)

### PR DESCRIPTION
**Pull request by eldang**
Addresses the first two points in https://github.com/eldang/ballard-4culture/issues/16

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,22 +79,7 @@
...
-			<div class='legend'>
-				<div class='legend-title'>Click on a layer name below to toggle its visibility</div>
-				<div class='legend-scale'>
-					<ul class='legend-labels'>
-						<li onClick="showHideLayer('people-points', markerNames=['people_locations']);"><span id="raising_blended_learners_campuses"></span>People</li>
-						<li onClick="showHideLayer('raising-school-leaders-points', markerNames=['raising_school_leaders']);"><span id="raising_school_leaders"></span>Places</li>
-					</ul>
-
-
-				</div> <!-- end of div class='legend-scale' -->
```
Comment (eldang) [inline index.html:92]: This code has been commented out for some time and seems unneeded in this project

File: index.html [hunk 2]
```diff
@@ -79,22 +79,7 @@
...
 
-				<div class='legend-source'>Source: <a href="https://www.ballardhistory.org/" target="_blank">Ballard Historical Society</a></div>
-				<div class='map-credit'>Map design by <a href="http://www.coregis.net/" target="_blank">CORE GIS</a></div>
-			</div> <!-- end of div id="legend" -->
-		</div> <!-- end of div id="main" -->
```
Comment (eldang) [inline index.html:97]: I think this code was accidentally commented out, but lines 62-67 above have since replaced it.

**Comment by eldang** [comment]
I recommend merging this PR after all the others I've left open, because it has some potential for merge conflicts and does not need to block any other work so if a conflict does arise it's going to be no trouble to wait for me to resolve it.

**Review by coregis — APPROVED** [review]

**Comment by coregis** [comment]
Thank you @eldang for sorting out the order in which to merge all these PRs, it seems to have worked flawlessly.

**Pull request merged**